packages/protocol/contracts/signal/SignalService.sol

```diff
@@ -39,13 +40,13 @@ contract SignalService is AuthorizableContract, ISignalService {
     struct Hop {
         address relayerContract;
         bytes32 stateRoot;
-        bytes[] merkleProof;
+        bytes merkleProof; // Merkle proof consists of account proof and storage proof encoded (concatenated) together.
```

Comment:
Can we move this comment to the top and keep one copy only, such that when we say merkleProof in this file, we means the whole proof.